# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gdmap/01-manpage.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Description: Correct the "whatis" information in the manpage
Author: Mario Iseli <mario@debian.org>

--- gdmap-v1.3.1/data/gdmap.1.vanilla	2025-06-28 11:05:25.222805165 +0200
+++ gdmap-v1.3.1/data/gdmap.1	2025-06-28 11:05:43.427121629 +0200
@@ -1,6 +1,6 @@
 .TH GDMAP 1 "16 Oct 2005"
 .SH NAME
-\fBGraphical Disk Map
+gdmap -\ Graphical Disk Map
 
 .SH SYNOPSIS
 gdmap [\fB\-?\fR|\fB--help\fR]
